Tools/GNUMake/comps/pgi.mak

```diff
@@ -82,11 +82,7 @@ else
 endif
 
 # The logic here should be consistent with what's in nvcc.mak
-ifeq ($(shell expr $(gcc_major_version) \>= 5), 1)
```

Comment:
is this correct to check GCC's version here? Does PGI use their stdlib and we rely on that? Does not mean the language features are there, though.

Reply:
That code came from @atmyers in commit daa5a07 based on the git log, maybe he can comment on his reasoning for it.

Reply:
PGI uses some gcc headers.  If those headers are C++11, pgc++ -std=c++14 will not work.  I remember at some point we had to do that to be able to compile on both ornl machines and our local machines.  We were not happy with the check.  But didn't find another way.

Reply:
To put it another way.  The issue was a version of pgc++ can support C++14 doesn't mean `pgc++ -std=c++14` will always work.  It actually depends on the gcc headers they use.

Reply:
Should we check if the GCC major is a certain minimum and the PGI major is sufficiently high as well?